pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -505,10 +507,10 @@ func (r *ReconcileBareMetalHost) actionMatchProfile(prov provisioner.Provisioner
 func (r *ReconcileBareMetalHost) actionProvisioning(prov provisioner.Provisioner, info *reconcileInfo) (result reconcile.Result, err error) {
 	var provResult provisioner.Result
 
-	getUserData := func() (string, error) {
+	getUserData := func() (string, map[string]interface{}, error) {
```

Comment:
Why is userData a string but networkData an interface{}?

Reply:
done switched to []byte